src/board/processor.ts

```diff
@@ -5,7 +5,8 @@ import { END_OF_CODE } from 'instruction/special'
 import { Word } from 'types/binary'
 import { EventEmitter } from 'types/events/emitter'
 
-const cycleSpeed: number = 10
+// We want the processor to cycle as fast as possible.
```

Comment:
Maybe you could add the comment from your PR also here that 0 causes the engine to execute the method on the next event loop cycle.